spec/models/patient_spec.rb

```diff
@@ -5,14 +5,14 @@
   subject(:patient) { build(:patient) }
 
   describe 'Associations' do
-    it { should belong_to(:address).optional }
-    it { should have_many(:phone_numbers) }
-    it { should have_many(:blood_pressures) }
-    it { should have_many(:blood_sugars) }
-    it { should have_many(:prescription_drugs) }
-    it { should have_many(:facilities).through(:blood_pressures) }
-    it { should have_many(:appointments) }
-    it { should have_one(:medical_history) }
+    it { is_expected.to belong_to(:address).optional }
```

Comment:
Is this some newfangled rspec syntax?

Reply:
It's the equivalent of replacing `object.should eq(something)` with `expect(object).to eq(something)`. The `expect` syntax is the new recommended way, and `should` is frowned upon (it does a bunch of `Object` monkeypatching I think to power its DSL)